pkg/controller/plan/adapter/openstack/builder.go

```diff
@@ -229,9 +229,14 @@ func (r *Builder) PodEnvironment(_ ref.Ref, _ *core.Secret) (env []core.EnvVar,
 func (r *Builder) PersistentVolumeClaimWithSourceRef(da interface{}, storageName *string, populatorName string, accessModes []core.PersistentVolumeAccessMode, volumeMode *core.PersistentVolumeMode) *core.PersistentVolumeClaim {
 	image := da.(*openstack.Image)
 	apiGroup := "forklift.konveyor.io"
-	size := image.VirtualSize
+	virtualSize := image.VirtualSize
+	// virtual_size may not always be available
```

Comment:
would it make sense to add a validation warning about this?

Reply:
As discussed, we use the size provided by the user anyway